session_items.py

```diff
@@ -1,11 +1,16 @@
 from flask import session
```

Comment:
You can delete this file (session_items.py) now

Reply:
Please check the changes with commit ID - ad8f0c427b592c542b116bdb03e61ebd8ac26d80